GNUmakefile

```diff
@@ -22,3 +22,30 @@ help:
 .PHONY: test
 test: ## Run all tests
 	$Q $(CURDIR)/test.sh --type oss --unprivileged false --latest-njs false
+
+# Check if the 'open' command exists on the system
+OPEN := $(shell command -v open 2> /dev/null)
+
+# Define the open command based on availability
+ifeq ($(OPEN),)
+	OPEN_COMMAND = xdg-open
+else
+	OPEN_COMMAND = open
+endif
+
+docs_destination_directory = "reference"
+
+.PHONY: docs
```

Comment:
**Annotation**:
This may not be the correct thing to do, but without it I could not get the `make jsdoc` `make clean` `make jsdoc` flow to work correctly.

I figure it's safer to build the docs every time anyway.  In a CI context this shouldn't matter.